app/views/notify_user/file_uploaded.html.haml

```diff
@@ -0,0 +1,6 @@
+%h2 A file was uploaded.
+
+%p
+  A file was uploaded for the following client. 
+%p
+  = link_to "Client ##{@client.id}", client_files_url(@client)
```

Comment:
This will need to link to the window version of files, since most people don't have access to the client section.

Reply:
Might even be worth looking at the user access to determine which link to use.  If you don't have access to either `can_manage_client_files` or `can_manage_window_client_files`, then you shouldn't get a notification.